ow_faults/src/FaultInjector.cpp

```diff
@@ -26,8 +29,10 @@ FaultInjector::FaultInjector(ros::NodeHandle node_handle)
   m_dist_pitch_ft_sensor_pub = node_handle.advertise<geometry_msgs::WrenchStamped>(ft_sensor_dist_pitch_str, 10);
 
   //power fault publishers and subs
-  m_fault_power_state_of_charge_pub = node_handle.advertise<std_msgs::Float64>("temporary/power_fault/state_of_charge", 10);
-  m_fault_power_temp_pub = node_handle.advertise<std_msgs::Float64>("temporary/power_fault/temp_increase", 10);
+    // will return to checking state of charge once that returns
+  m_power_soc_sub = node_handle.subscribe("/power_system_node/state_of_charge", 1000, &FaultInjector::powerSOCListener, this); 
+  m_power_temp_sub = node_handle.subscribe("/power_system_node/battery_temperature", 1000, &FaultInjector::powerTempListener, this); 
```

Comment:
Curious if there's a specific reason why the queue size is so large (1000).  I've typically used and seen values < 10.

Reply:
that was the prior publish rate from the power node, but I can double check on if that's changed.

Reply:
My understanding of the queue size is that is how many messages are allowed to queue up.   Not sure how this relates numerically to the publish rate of the publisher, but would like to understand it better...